fdc.js

```diff
@@ -146,7 +148,7 @@ export function BaseDisc(fdc, name, data, flusher) {
             case 6:
                 this.fdc.discFinishRead();
                 this.rsector++;
-                if (this.rsector === 10) this.rsector = 0;
+                if (this.rsector === (this.sectorsPerTrack)) this.rsector = 0;
```

Comment:
super nitpick (and I'm surprised the `prettier` thing didn't pick up on this) - would you mind removing the extra parens here?

In fact, I may have broken the pre-commit hooks; can you run:
```
$ npx prettier --write .
```

(or similar) and see if it fixes up/complains :) thanks!

Reply:
Yes I wasn't sure at what point all the pretty stuff was going to happen, didn't seem to launch on pre-commit so have run it manually.